build.gradle

```diff
@@ -64,10 +63,10 @@ dependencies {
 	}
 	implementation 'net.sf.jopt-simple:jopt-simple:5.0.3'
 
-	testCompileClasspath 'org.jetbrains:annotations:19.0.0'
+	testCompileClasspath 'org.jetbrains:annotations:20.1.0'
```

Comment:
Wouldn’t be 

> testImplementation 

better here?

Reply:
Because it’s the same as 

> testCompileOnly

Reply:
No, `testImplementation` and `testCompileOnly` have different behaviours. (`testImplementation` is also available at runtime, while the other one is only at compile time)